pom.xml

```diff
@@ -84,9 +84,7 @@ flexible messaging model and an intuitive client API.</description>
 
     <pulsar.client.python.version>3.4.0</pulsar.client.python.version>
 
-    <UBUNTU_MIRROR>http://archive.ubuntu.com/ubuntu/</UBUNTU_MIRROR>
```

Comment:
nit: why do we need to remove UBUNTU_MIRROR?

Reply:
That was removed in this PR as a cleanup. It was needed before 3.3.x with the Ubuntu  Image.